oai_agents/common/population.py

```diff
@@ -116,7 +116,7 @@ def save_population(args, population):
             seed=None,
         )
         rt.agents = population[layout_name]
-        rt.save_agents(tag='aamas25')
+        rt.save_agents(tag='last')
```

Comment:
can you use [this](https://github.com/HIRO-group/multiHRI/blob/1d8295dc6cc149a498b7e21eb7187139454b1cab/oai_agents/common/tags.py#L84) instead: CheckedPoints.FinalTrainedModel

If you could also come up with a better name for CheckedPoints that would be great